File: linguagem-java/emerson_moreira/java/MainSearch.java
class Main {
  public static void main(String[] args) {
    Search sear = new Search();

    int array[] = {10, 20, 80, 30, 60, 50, 
                     110, 100, 130, 170};
    int num = 100;
    int result = sear.SearchLine(array, num);

    System.out.println("O número "+num+" esta na posição "+result+" do array");
  }
}

File: liguagem-php/emerson_moreira/php/mainSearch.php
<?php
include ("search.php");

$search = new SearchArray();

$arr = array(4, 2, 5, 6, 14, 7, 15, 3);
$num = 14;
echo "O numero ",$num, " esta na posição ",$search->search_index($arr, $num), " do array.";